app/views/votes/_voting.html.erb

```diff
@@ -1,11 +1,10 @@
 <% if @user_votes.include? vote %>
-  <% Vote::VOTE_TYPES.each do |reason, value| %>
+  <% t(:votes).each do |value, reason| %>
     <%= link_to reason, quote_vote_path(quote, vote, { vote: { value: value } } ), method: :patch %>
   <% end %>
   <%= link_to "Erase my vote", quote_vote_path(quote, vote), method: :delete %>
 <% else %>
-  <% Vote::VOTE_TYPES.each do |reason, value| %>
+  <% t(:votes).each do |value, reason| %>
```

Comment:
Storing objects in the locales isn't a conventional thing to do. Try a different approach.